crates/volta-core/src/error/details.rs

```diff
@@ -502,7 +503,7 @@ for {}
                 "Could not create directory {}
 
 Please ensure that you have the correct permissions.",
-                dir
+                dir.to_string_lossy()
```

Comment:
We should probably prefer [`PathBuf::display()`](https://doc.rust-lang.org/1.34.1/std/path/struct.PathBuf.html#method.display) here (and throughout) instead, since it's explicitly designed for this:

> Returns an object that implements `Display` for safely printing paths that may contain non-Unicode data.

Reply:
Done.